.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -219,7 +219,7 @@ swap:
   perfcounter: performance counter
   perimeter network: DMZ
   plaintext|plain-text|cleartext|clear text: plain text
-  plugin|plugins|plug in|plug ins: plug-in # https://www.merriam-webster.com/dictionary/plug-in
+  plug in|plug ins: plugin # https://www.merriam-webster.com/dictionary/plug-in
```

Comment:
Nit: Can we remove this comment? We don't need it here and it's confusing.

Reply:
@aireilly - Good catch. Thanks. Funny how my brain has been trained to totally ignore stuff that's remmed out!
Noise removed :-)

Reply:
Merged :)